_posts/2021-03-29-new-release-2021-1.md

```diff
@@ -0,0 +1,11 @@
+---
+title:  "Connector SDK now targets Tableau 2021.1"
+abstract: "The master SDK branch now targets Tableau 2021.1. The Connector SDK for 2020.4 has released for those wishing to target that version."
+---
+
+On March 29th, with the full release of Tableau 2021.1 we moved the latest version of the master branch to target Tableau 2020.1. This means that the samples and connectors made with this version of the SDK may not work with older versions of Tableau. To develop against Tableau 2020.4, please download the [Tableau Connector SDK for Tableau 2020.4](https://github.com/tableau/connector-plugin-sdk/releases/tag/tableau-2020.4) release.
+
+- __OAuth Authentication Support__ <br/>
+ Starting in Tableau 2021.1 a connector using Connection Dialog V2 style, OAuth is supported as one of the categories for authetication. To learn more please refer  [here]({{ site.baseurl }}/docs/oauth). For a complete example please refer to [here](https://github.com/tableau/connector-plugin-sdk/tree/dev-2021.1/samples/scenarios/snowflake_oauth).
```

Comment:
Update link to use master now that it is 21.1

Reply:
Updated the link to reflect master branch